packages/components/src/CONTRIBUTING.md

```diff
@@ -0,0 +1,80 @@
+
+## Component refinement
+
+Before a component is published it will need to be fine-tuned:
+
+1. **Expand** to a minimally sufficient feature set. Resolve what features should be included in the component.
+2. **Reduce** scope to omit contentious features. Review proposed features and prune unessential items lacking consensus.
+3. **Quality assurance**: each contribution must be normalized into system standards.
+
+### Quality assurance
+
+To ensure quality, each component should be tested. The testing process should be done during the development of the component and before the component is published. 
+
+- **Accessibility**: Has the design and implementation accounted for accessibility? Please use the [WordPress accessibility guidelines](https://make.wordpress.org/accessibility/handbook/best-practices/).
+- **Visual quality**: Does the component apply visual style — color, typography, icons, space, borders, and more — using appropriate variables, and does it follow [visual guidelines](https://make.wordpress.org/design/handbook/design-guide/)?
+- **Responsiveness**: Does it incorporate responsive display patterns and behaviors as needed? Is the component designed from a mobile-first perspective? Do all touch interactions work as expected?
+- **Sufficient states & variations**: Does it cover all the necessary variations (primary, secondary, dense, etc.) and states (default, hover, active, disabled, loading, etc.), given the intended scope?
+- **Content resilience**: Is each dynamic word or image element resilient to too much, too little, and no content at all, respectively? How long can labels be, and what happens when you run out of space?
+- **Composability**: Does it fit well when placed next to or layered with other components to form a larger composition?
+- **Functionality**: Do all behaviors function as expected?
+- **Browser support**: Has the component visual quality and accuracy been assessed across Safari, Chrome, Firefox, IE, and other browsers across relevant devices? Please adhere to our [browser support requirements](https://github.com/WordPress/gutenberg/blob/master/packages/browserslist-config/index.js).
+- **Documentation**: Ensure that the component has proper documentation for development, design, and accessibility.
+
+## Deprecation
+
+A component/feature may need deprecation if:
+
+- There’s a preferred alternative or it’s no longer in demand
+- It’s no longer necessary 
+- It’s moved to a different location
+
+At no point does the deprecated component become unavailable. Instead, the commitment to support it stops. The following steps are needed to deprecate a component:
+
+1. Communicate intent via regular channels.
+    1. Describe reasoning for deprecation.
+    2. Decide on a timeline.
+2. Ensure backwards compatibility to keep it available.
+3. Clearly mark the component as deprecated in documentation and other channels.
```

Comment:
💯 

Maybe get a bit more explicit here about the "other channels"—we want to mark as deprecated in docs, in Figma, maybe output a console warning or some such for devs? What else?